book/src/intro.md

```diff
@@ -284,28 +283,33 @@ SELECT avg(radius) as R, avg(width) as W FROM (
 {R:1.75,W:1.5}
 ```
 
-Things get more interesting when you want to do different types of processing
+Things get more interesting when you want to do different kinds of processing
```

Comment:
I figured this synonym was a slight improvement since the word "type" is already used in the technical sense close to here.